services/core-web/src/components/Forms/GeneratePermitForm.js

```diff
@@ -0,0 +1,229 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Field, reduxForm } from "redux-form";
+import { Form, Button, Col, Row, Popconfirm } from "antd";
+import { required } from "@common/utils/Validate";
+import { resetForm } from "@common/utils/helpers";
+import * as FORM from "@/constants/forms";
+import { renderConfig } from "@/components/common/config";
+
+const propTypes = {
+  cancelGeneration: PropTypes.func.isRequired,
+  handleSubmit: PropTypes.func.isRequired,
+  onSubmit: PropTypes.func.isRequired,
+  returnToPrevStep: PropTypes.func.isRequired,
+  documentList: PropTypes.arrayOf(PropTypes.any),
+  submitting: PropTypes.bool.isRequired,
+  isAmendment: PropTypes.bool.isRequired,
+};
+
+export const GeneratePermitForm = (props) => (
+  <Form layout="vertical" onSubmit={props.handleSubmit}>
+    <h2 className="padding-large--top">Part One: General Information</h2>
+    <Row gutter={32}>
+      <Col xs={24} md={12}>
+        <Field
+          id="mine_no"
+          name="mine_no"
+          label="Mine Number"
+          required
+          component={renderConfig.FIELD}
+          validate={[required]}
+          disabled={true}
```

Comment:
you can just pass "disabled", by having the prop defined it is already true. Not blocking, but good to know for the future.

Reply:
Changed